--- gpl-linux/verdep.h.orig     2016-02-02 17:08:12.425237356 +0000
+++ gpl-linux/verdep.h  2016-02-02 17:08:44.920465904 +0000
@@ -316,7 +316,7 @@
 #   ifdef REDHAT_RHEL53
 #     define POSIX_SETLEASE(fP, arg, flPP)  __setlease(fP, arg, flPP)
 #   else
-#     define POSIX_SETLEASE(fP, arg, flPP)  generic_setlease(fP, arg, flPP)
+#     define POSIX_SETLEASE(fP, arg, flPP)  generic_setlease(fP, arg, flPP, NULL)
 #   endif
 # else
 #     define POSIX_SETLEASE(fP, arg, flPP)  0
